platform/ios/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 ### Other
 
 * Fixed an issue where `-[MGLMapView metersPerPixelAtLatitude:]` was removed, but not marked as unavailable. ([#11765](https://github.com/mapbox/mapbox-gl-native/pull/11765))
+* Fixed an issue where selecting an onscreen annotation could move the map (seen in [ios-sdk-examples](https://github.com/mapbox/ios-sdk-examples)). ([#11731](https://github.com/mapbox/mapbox-gl-native/pull/11731))
```

Comment:
Couple small suggestions:

- I don’t think we need to mention `ios-sdk-examples`.
- Selecting an annotation should move the map in certain circumstances, so perhaps we should clarify that the movement was undesirable/inadvertent.